MAUI/ImageEditor/shape.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Shape Annotation in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about the shape annotation feature of Syncfusion .NET MAUI ImageEditor(SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Shape Annotation in .NET MAUI ImageEditor (SfImageEditor)
+
+The image editor control allows you to add various shapes with customizable settings
+
+## Shape types
+
+* Circle
+* Rectangle
+* Arrow
+* Line
+* Dotted
+* DoubleArrow
+* DottedArrow
+* DottedDoubleArrow
+
+## Customize shape settings
+
+The shape annotations can be customized using common annotation customization properties.
+
+* `Bounds`: Specifies the bounds of the annotation view.
+* `AllowDrag`:  Enables or disables the dragging for annotations.
+* `AllowResize`: Enables or disables the resizing for annotations.
+* `Opacity`: Specifies the opacity of annotations.
+
+Shapes can be inserted and customized by changing their stroke color, fill color, and stroke thickness. You can create shapes using either a toolbar or the `AddShape` method.
```

Comment:
Remove

Shapes can be inserted and customized by changing their stroke color, fill color, and stroke thickness. You can create shapes using either a toolbar or the `AddShape` method.

Reply:
Removed